### DISCUSSION TITLE
Can I filter results by file type, f.e. Word/Excel/Powerpoint?

### DISCUSSION THREAD

**Discussion by sliziky** [microsoft-search/pnp-modern-search#2485] [Q&A]
Hi,

I know there is a possibility to filter results by file type, which means I see doc, docx, xls, xlsx, etc.. as the options, so if I want to see all Word documents then I have to select both doc and docx in the filter.

I would like to "merge" these file extensions and filter just by selecting "Word document" (doc, docx), "Excel document" (xls, xlsx), "Text document" (txt) etc.. is this possible or not? 

Thanks for any tips :)

**Comment by wobba** [comment]
See this PR which is in progress https://github.com/microsoft-search/pnp-modern-search/pull/1845

**Reply by sliziky** [reply]
This looks great! Is this feature coming with v4.8? When do you think it is going to be released? :)

**Reply by edarroudi** [reply]
Hey @sliziky I started to move on with this PR, once I'll get feedback from the maintainers I'll finish this quickly. Just don't know if it's going to make it in the next release - if not you woud either have to wait about 3 months or maybe pull the dev version once the PR is approved and use that on your tenant.

**Comment by kasperbolarsen** [comment]
Pretty sure this have been solved, and if you need more then there is https://github.com/microsoft-search/pnp-modern-search-layouts/tree/main/Refiner/File%20Extensions%20icons
